javascript/ql/src/semmle/javascript/Functions.qll

```diff
@@ -158,6 +158,45 @@ class Function extends @function, Parameterized, TypeParameterized, StmtContaine
     result = getAReturnStmt().getExpr()
   }
 
+  /** 
+   * Gets a `ConcreteControlFlowNode` which corresponds to an executable portion of
```

Comment:
Could you try simplifying this a little? I realise you want it to be technically accurate, but one of the most important uses of these comments is that they are shown during autocomplete, and for that purpose it's better if the first sentence is short and easy to grasp intuitively.

You can then add a more precise technical definition as a second (third, ...) paragraph, which won't be shown as part of the autocomplete hover.